tests/test_chained_calls.py

```diff
@@ -63,7 +63,7 @@ def test_chained_inference_exact_match(atol_inference=1e-4):
     inputs = torch.randn(1, 8, config.hidden_size)
 
     outputs_inference = []
-    with remote_block.inference_session() as sess:
+    with remote_block.inference_session(max_length=inputs.shape[1]) as sess:
```

Comment:
Maybe add a test checking what happens if we exceed `max_length`?

Reply:
Done, see test_block_exact_match.py